packages/format-library/src/underline/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { toggleFormat } from '@wordpress/rich-text';
+import { RichTextToolbarButton, RichTextShortcut } from '@wordpress/editor';
+
+const name = 'core/underline';
+
+export const underline = {
+	name,
+	title: __( 'Underline' ),
+	tagName: 'u',
```

Comment:
Let's use a `span` with inline styles, just as we did in the classic editor.

> This element used to be called the "Underline" element in older versions of HTML, and is still sometimes misused in this way. To underline text, you should instead apply a style that includes the CSS text-decoration property set to underline.
>
> https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u

Reply:
I updated the code to use an inline style, thank you for pointing this. Now the markup is exactly the same as the one used by the classic editor.